### DISCUSSION TITLE
what does recoil_early_rendering_2021  mean ?

### DISCUSSION THREAD

**Discussion by huntye1** [facebookexperimental/Recoil#1197] [Q&A] [answered]
i see source code like:   `gkx('recoil_early_rendering_2021')`
what does this mean?

**Comment by drarmstr** [comment, marked as answer]
`gkx()` checks if a feature flag is enabled.  In this case `recoil_early_rendering_2021` flag enables support for `useTransition()` and syncing Recoil and React state changes.  See #759 and #1076
